packages/api/src/entity/library_item.ts

```diff
@@ -204,4 +204,16 @@ export class LibraryItem {
 
   @Column('text')
   highlightAnnotations?: string[]
+
+  @Column('timestamptz')
+  seenAt?: Date
+
+  @Column('text')
+  topic?: string
```

Comment:
Should we do this as a tree? Probably fine either way for now.

Reply:
Yep, I forget to change this 😢